packages/apps/desktop-viewer-test/src/frontend/app/ITwinViewerApp.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import {
```

Comment:
please follow our `imports` style and not use one-per-line. Are you using our lint rules?

Reply:
This repo uses prettier for formatting. Currently, it is using the default printWidth of 80 chars which will wrap long imports. I will update that to 120 in a separate PR since it will change pretty much every file and that should help with these (though it will not remove all of them). 

I've also setup a meeting to discuss unifying this and other open source repo linting/formatting rules and will reformat/refactor based on the outcome of that meeting